login/login_api.go

```diff
@@ -20,6 +20,22 @@ func NewLoginAPI(p LoginService) LoginAPI {
 	return LoginAPI{LoginService: p}
 }
 
+// FindSamePassword ...
+func (p *LoginAPI) FindSamePassword(c *gin.Context) {
+
+	password := c.PostForm("password")
```

Comment:
We use JSON in HTTP request body between clients and server commonunication. Can you change this to:
```

...
var password Password
c.BindJSON(&password)
...
```

Reply:
I just completed [c685435](https://github.com/pass-wall/passwall-server/pull/54/commits/c685435596dfa79e34c515a2ce9878bf517f96c4)